guides/common/modules/proc_registering-a-host-without-puppet.adoc

```diff
@@ -5,7 +5,7 @@ By default, the bootstrap script configures the host for content management and
 If you have an existing configuration management system and do not want to install Puppet on the host, use `--skip-puppet`.
 
 .Procedure
-* On {EL} 8, enter the following command:
+* On {EL} 9 or 8, enter the following command:
```

Comment:
Is there a stylistic reason for using "9 or 8" instead of "8 or 9"?

Reply:
We tend to list versions from latest to oldest.

Reply:
Why do you list from latest to oldest?

Reply:
ISG does not provide a specific guideline about version order but they list versions from earlier to later in their examples. See [References to multiple releases](https://www.ibm.com/docs/en/ibm-style?topic=references-product-versions#references-to-multiple-releases).